core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.directive.html

```diff
@@ -23,7 +28,7 @@ <h3>Add Misconception</h3>
         </div>
         <div>
           <input class="form-control protractor-test-misconception-name-field" type="text" ng-model="misconceptionName" maxlength="<[MAX_CHARS_IN_MISCONCEPTION_NAME]>" autofocus>
-          <span ng-if="misconceptionName.length === MAX_CHARS_IN_MISCONCEPTION_NAME" class="form-text" style="color: red; font-size: smaller">
+          <span ng-if="misconceptionName.length === MAX_CHARS_IN_MISCONCEPTION_NAME" class="form-text oppia-form-text">
```

Comment:
Ditto: Use common class in oppia.css for "font-size: smaller"; Use text-danger for "color: red";